python/lsst/alert/packet/schema.py

```diff
@@ -428,3 +430,12 @@ def from_file(cls, filename=None):
                                      if schema['name'] == root_name)
 
         return cls(schema_definition)
+
+    def get_schema_id(self):
+        """Retrieve the schema id used in the schema registry.
+        """
+        numbers = re.findall(r'\d+', self.definition['name'])
```

Comment:
maybe we should add an `assert(len(numbers) == 2)` here?  I'm nervous about having versioning driven by regexp.

Reply:
Added